### DISCUSSION TITLE
[Feature request] Editable comment/description for a table or database

### DISCUSSION THREAD

**Discussion by akzarma** [StarRocks/starrocks#19269] [Announcements]
I found that we can set the comment for a table while creating it but we cannot update it anymore once it is created. (tried with ALTER TABLE command but it didn't work)

So, here are mainly 2 feature requests:
1. Make the comment of a table editable
2. Add the functionality to add a comment on a database too.

**Comment by alberttwong** [comment]
Can you create an issue ticket for this?
